chamber/models.py

```diff
@@ -107,6 +115,13 @@ def eval_props(self):
         self.rho_m = 1 / HAPropsSI('Vha', 'T', ref_temp, 'Y', ref_x,
                                    'P', self.pressure)
 
+        self.nu_m = self.mu_m/self.rho_m
+        self.alpha_m = self.k_m/(self.cp_m * self.rho_m)
+        self.beta_m = 1/ref_temp
+        self.beta_star_m = (const.M2 - const.M1)/(ref_x*const.M1*(const.M2 - const.M1)/(ref_x*const.M1 + (1-ref_x)*const.M2) + const.M1)
+
+        self.Ra_number = const.GRAVIT_ACCLR*(self.beta_m*(self.temp_s - self.temp_e) + self.beta_star_m*(self.m_1s - self.m_1e))*pow(self.length, 3)/(self.alpha_m*self.nu_m)
+
```

Comment:
This is ok for now, but what we will want to do is make a new class method, that will calculate all of the dimensionless numbers one the properties have been evaluated. We will merge this in now, but that will be the next thing you work on.